k8s/demo/common/rpe/draft-store-service.yaml

```
@@ -0,0 +1,33 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: draft-store-service-demo
+  namespace: rpe
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: draft-store-service-demo
+  timeout: 900
+  forceUpgrade: true
```

May be not needed, to be aligned with other envs
```suggestion
  forceUpgrade: true
```